src/uu/install/src/install.rs

```diff
@@ -750,6 +753,9 @@ fn copy_normal_file(from: &Path, to: &Path) -> UResult<()> {
 fn copy_file(from: &Path, to: &Path) -> UResult<()> {
     // fs::copy fails if destination is a invalid symlink.
     // so lets just remove all existing files at destination before copy.
+    if to.is_dir() && !from.is_dir() {
+        return Err(InstallError::OverrideDirectoryFailed(to.to_path_buf().clone()).into());
+    }
```

Comment:
The comment belongs to the existing code and not to the new code.
```suggestion
    if to.is_dir() && !from.is_dir() {
        return Err(InstallError::OverrideDirectoryFailed(to.to_path_buf().clone()).into());
    }
    // fs::copy fails if destination is a invalid symlink.
    // so lets just remove all existing files at destination before copy.
```